src/diagnostics_component/utils/file_info_utils/tests/file_info_utils_ut.cpp

```diff
@@ -38,7 +38,10 @@ TEST_CASE("FileInfoUtils_FillFileInfoWithNewestFilesInDir")
             testCandidateLastWrite));
 
         CHECK(strcmp(sortedLogFileArray[0].fileName, testCandidateFileName) == 0);
+
+        free(sortedLogFileArray[0].fileName);
```

Comment:
this test is CPP.  use unique_ptr

Reply:
i see we have :

calloc_wrapper
FreeDeleter (unused?)
std::unique_ptr<char, decltype(&free)> managed(ptr, free);
and i think other wrappers

we should converge on one.  and use it.

Reply:
I moved to using calloc_wrapper / cstr_wrapper and got rid of all the reset from ut .cpp files.